includes/class-newspack-ads-gam.php

```diff
@@ -611,6 +612,18 @@ private static function get_line_items() {
 		return $line_items;
 	}
 
+	/**
+	 * Get all line items given an order ID.
+	 *
+	 * @param int $order_id Order ID.
+	 *
+	 * @return LineItems[] Array of LineItems.
+	 */
+	public static function get_line_items_by_order_id( $order_id ) {
+		$statement_builder = ( new StatementBuilder() )->where( sprintf( 'orderId = %d', $order_id ) );
```

Comment:
I don't think so, it'd be harder to maintain since we would have to clear the previously set clauses on every use. Instantiating on each use shouldn't have an impact on performance, I think.